lib/config/fileconf.go

```diff
@@ -1100,3 +1104,33 @@ func (o *OIDCConnector) Parse() (services.OIDCConnector, error) {
 	}
 	return v2, nil
 }
+
+// Metrics is a `metrics_service` section of the config file:
+type Metrics struct {
+	// Service is a generic service configuration section
+	Service `yaml:",inline"`
+
+	// MTLSFlag states whether the metrics endpoint should be secured with mTLS
+	// or if metrics should be shipped in clear text
+	MTLSFlag string `yaml:"mtls,omitempty"`
```

Comment:
Is this flag needed at all? Can we just auto-enable mTLS if key pairs and/or CAs are provided?

Reply:
that was the original design if I remember correctly.
it was proposed in the rfd discussion https://github.com/gravitational/teleport/pull/6469

Reply:
Are there any downsides to enabling mtls just based on the presence of keypairs/CA? The upside is one less configuration field. @awly @russjones what do you folks think?

Reply:
Removing the boolean field makes sense to me

Reply:
sounds good